core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/ValidateRequestBuilder.java

```diff
@@ -28,20 +29,28 @@
 public final class ValidateRequestBuilder 
 		extends BaseRequestBuilder<ValidateRequestBuilder, BranchContext, ValidationResult>
 		implements RevisionIndexRequestBuilder<ValidationResult> {
-
+	
 	ValidateRequestBuilder() {}
 	
 	private Collection<String> ruleIds;
 	
+	private Map<String, Object> ruleParameters;
+	
 	public ValidateRequestBuilder setRuleIds(Collection<String> ruleIds) {
 		this.ruleIds = ruleIds;
 		return getSelf();
 	}
 	
+	public ValidateRequestBuilder setFilterOptions(Map<String, Object> ruleParameters) {
```

Comment:
Please change the method name here and the parameter name in the ValidateRequest class as well.